microservices/realtime/src/Services/RegistrationChannelResolver.php

```diff
@@ -32,18 +32,53 @@ public function __construct(
         $this->ddiProviderRepository = $ddiProviderRepository;
     }
 
+    public function getDefaultCriteria(array $tokenPayload): array
```

Comment:
Type hint for argument is not needed here?

Reply:
Done